config/jobs/kubernetes-sigs/image-builder/image-builder-presubmits.yaml

```diff
@@ -20,4 +20,29 @@ presubmits:
             cpu: 1000m
     annotations:
       testgrid-dashboards: sig-cluster-lifecycle-image-builder
-      testgrid-tab-name: pr-azure-all
+      testgrid-tab-name: pr-azure-vhds
+  - name: pull-azure-sigs
+    labels:
+      preset-azure-cred: "true"
+    decorate: true
+    always_run: false
+    optional: true
+    decoration_config:
+      timeout: 1h
+    max_concurrency: 5
+    path_alias: sigs.k8s.io/image-builder
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20201030-4553f14-master
```

Comment:
should this be: v20201031-122dc79-master (compared to the other jobs)?

Reply:
it probably got updated since I pulled... I'll update it but shouldn't matter since it gets automatically updated by the bot